spec/jobs/nb_person_push_job_spec.rb

```diff
@@ -3,9 +3,18 @@
 RSpec.describe NbPersonPushJob, type: :job do
   describe "#perform" do
     it "calls NationBuilder with correct args" do
-      args = { attributes: { email: "user@example.com", phone: '510-555-9999' } }
+      args = { attributes: { email: 'user@example.com', phone: '510-555-9999' } }
       expect(Integration::NationBuilder).to receive(:create_or_update_person).with(args)
-      NbPersonPushJob.new.perform("email"=>"user@example.com", "phone"=>"510-555-9999")
+      NbPersonPushJob.new.perform(email: 'user@example.com', phone: '510-555-9999')
+    end
+
+    it "creates RSVP when event_id is present" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.